src/main/java/entity/Message.java

```diff
@@ -66,59 +67,59 @@ public Message(long messageId, long chatId, long senderId, Long attachmentId) {
         this.attachmentId = attachmentId;
     }
 
-    public void setMessageId (long messageId) {
+    public void setMessageId(long messageId) {
         this.messageId = messageId;
     }
 
-    public void setChatId (long chatId) {
+    public void setChatId(long chatId) {
         this.chatId = chatId;
     }
 
-    public void setSenderId (long senderId) {
+    public void setSenderId(long senderId) {
         this.senderId = senderId;
     }
 
-    public void setTextContent (String textContent) {
+    public void setTextContent(String textContent) {
         this.textContent = textContent;
     }
 
-    public void setAttachmentId (Long attachmentId) {
+    public void setAttachmentId(Long attachmentId) {
         this.attachmentId = attachmentId;
     }
 
-    public void setReceivedTs (Timestamp receivedTs) {
+    public void setReceivedTs(Timestamp receivedTs) {
         this.receivedTs = receivedTs;
     }
 
-    public Timestamp getCreationTs () {
+    public Timestamp getCreationTs() {
         return creationTs;
     }
 
-    public long getMessageId () {
+    public long getMessageId() {
         return messageId;
     }
 
-    public long getChatId () {
+    public long getChatId() {
         return chatId;
     }
 
-    public long getSenderId () {
+    public long getSenderId() {
         return senderId;
     }
 
-    public String getTextContent () {
+    public String getTextContent() {
         return textContent;
     }
 
-    public Long getAttachmentId () {
-        return attachmentId;
+    public Optional<Long> getAttachmentId() {
```

Comment:
Please use `OptionalLong` instead of `Optional<Long>`.

Reply:
Done. But could you explain why we are doing this? Couldn't understand the advantage of `OptionalLong` over `Optional<Long>`.

Reply:
`Optional<Long>` or `Optional<T>` in general, stores the data as an object. Which means, for primitive data types such as `long` or `int`, the data is wrapped in the wrapper types `Long` and `Integer` respectively.
That is why there are these specialised versions for primitive types, `OptionalLong`, `OptionalInteger` which stores them as primitives instead of wrapping them.